x-pack/plugins/security/public/authentication/login/login_page.tsx

```diff
@@ -67,6 +67,12 @@ const loginFormMessages: Record<LogoutReason, NonNullable<LoginFormProps['messag
       defaultMessage: 'Your session has timed out. Please log in again.',
     }),
   },
+  CONCURRENCY_LIMIT: {
+    type: LoginFormMessageType.Info,
+    content: i18n.translate('xpack.security.login.concurrencyLimitDescription', {
+      defaultMessage: 'You have logged in on another device. Please log in again.',
```

Comment:
Nit/question: As the number of concurrent sessions is configurable, should we be more explicit? e.g. "The number of allowable concurrent logins has been exceeded." or "You have logged in on another device, exceeding the concurrent session limit of N."
Not sure this is necessary, but thought I'd mention.

Reply:
I phrased this intentionally simple as I don't think the technical details / specific settings are important here for the user. As an administrator it might be of value but my thinking was that they can refer to the debug logs which contains more information about the exact logout reason if required.

Reply:
That seems reasonable to me. 👍